v4l2/v4l2_encode.cpp

```diff
@@ -58,6 +58,14 @@ bool V4l2Encoder::start()
     streamFormat.streamFormat = AVC_STREAM_FORMAT_ANNEXB;
     m_encoder->setParameters(VideoConfigTypeAVCStreamFormat, &streamFormat);
 
+    if (m_svct) {
+        VideoTemporalLayers layers;
+        layers.numLayersMinus1 = 1; // It is two layers actually
+        layers.bitRate[0] = 3000000;
```

Comment:
How about set a default bitrate according to the "m_videoParams.resolution" in case m_videoParams.rcParams.bitRate == 0. As far as I know, 4 Mb/s for 1080P.